### PR TITLE
Getting tipsy element size with getBoundingClientRect

### LINKED ISSUE #7256: Mapviews tooltip is not rendered in the correct position

### PR DESCRIPTION
**Pull request by xavijam**
There is a problem right now with the map views tooltip, and it is due to the calculation of the size of the element. We use within tipsy:

https://github.com/CartoDB/cartodb/blob/master/vendor/assets/javascripts/jquery.tipsy.js#L48

But for any reason, over svg elements, that function doesn't work properly (it returns 0 for width and height). Using `getboundingClientRect` fixes the bug.

Tested in several browsers (firefox, safari, chrome) and in several places.

#### Before:

screen shot 2016-04-27 at 10 30 59

#### After:

screen shot 2016-04-27 at 10 28 23
screen shot 2016-04-27 at 10 28 01
screen shot 2016-04-27 at 10 27 50

CR: @javierarce 
cc @saleiva 
Fixes #7256.

### CODE REVIEW

**Comment by xavijam** [comment]
Checked in ded02 that the fix is working properly :).

File: vendor/assets/javascripts/jquery.tipsy.js [hunk 1]
```diff
@@ -45,8 +45,8 @@
 
             //  Fixed by Arce
```
Comment (javierarce) [inline vendor/assets/javascripts/jquery.tipsy.js]: Could we get rid of this incriminating evidence? :P

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cartodb-ui",
-  "version": "3.23.65",
+  "version": "3.23.66",
```
Comment (javierarce) [inline package.json]: This is now the current version of the front, so it should probably be bumped.

**Comment by javierarce** [comment]
👍  +2 minor comments

File: package.json (hunk 2 above)
Comment (xavijam) [inline package.json]: ok

File: vendor/assets/javascripts/jquery.tipsy.js (hunk 1 above)
Comment (xavijam) [inline vendor/assets/javascripts/jquery.tipsy.js]: :joy:

**Comment by xavijam** [comment]
Thanks!

**Comment by xavijam** [comment]
Passing frontend tests, merging! :)

**Pull request merged**